features/image-transformations/image-enhancement-and-color-manipulation.md

```diff
@@ -133,3 +133,117 @@ URL - [https://ik.imagekit.io/demo/lipstick_2M.png](https://ik.imagekit.io/demo/
+| From Color (`from`) | **Optional**.<br/><br/>This is used to specify a *from* color for the gradient.<br/><br/>RGBA Code (e.g. `FFAABB50`) is a color representation model where the first six characters must be the RGB Hex Code and the last two characters must be a number between `00` and `99`, which indicates the opacity level. For example, `00` represents an opacity level of `0.00`, `01` represents an opacity level of `0.01`, and so on. | `String` | An [SVG recognized color](https://www.w3.org/TR/SVG11/types.html#ColorKeywords) keyword name eg. `aqua`;<br/><br/>or a RGB Hex Code eg. `FF0000`;<br/><br/>or a RGBA Hex Code eg. `FF000040`; | `FFFFFF` |
+| To Color (`to`) | **Optional**.<br/><br/>This is used to specify a *to* color for the gradient.<br/><br/>RGBA Code (e.g. `FFAABB50`) is a color representation model where the first six characters must be the RGB Hex Code and the last two characters must be a number between `00` and `99`, which indicates the opacity level. For example, `00` represents an opacity level of `0.00`, `01` represents an opacity level of `0.01`, and so on. | `String` | An [SVG recognized color](https://www.w3.org/TR/SVG11/types.html#ColorKeywords) keyword name eg. `aqua`;<br/><br/>or a RGB Hex Code eg. `FF0000`;<br/><br/>or a RGBA Hex Code eg. `FF000040`; | `00000000` |
+| Stop Point (`sp`) | **Optional**.<br/><br/>This is used to dictate where the break point should be made between the `from` color and the `to` color. In simpler words, it is the point (along the direction vector of the gradient) at which the `to` color achieves it’s pure form i.e. without any blends.<br/><br/>When the number is less than or equal to 1 i.e. a floating point, it would be treated as a percentage value. For example, `0.25` means 25%, `0.1` means 10% and so on.<br/><br/>When the number is greater than 1, it would be treated as an absolute value (in pixels). For example, `100` would mean 100px and so on. | `Number` or `String` | **Number** - A floating point number between `0.01` and `0.99` or an integer greater than or equal to `1`.<br/><br/>**String** - An arithmetic expression like `ih_div_2`. Read more about arithmetic expressions [here](../arithmetic-expressions-in-transformations.md). | `1` |
+
+Multiple parameters can be configured by appending them with an underscore (`_`). They will be applied independent of the order in which they are specified.
+
+Example usage - `e-gradient-<linear-direction>_<from-color>_<to-color>_<stop-point>`, which would look like, `e-gradient-ld-top_right_from-FF000040_to-00FF0040_sp-0.9`.
+
+{% tabs %}
+{% tab title="Original Image" %}
```

Comment:
I don't think that's true. I've seen multiple places to mean a "starting point" and not an "absolutely unprocessed" image. Ref: https://docs.imagekit.io/features/image-transformations/image-enhancement-and-color-manipulation

Reply:
For `e-sharpen`, this title is used, which I like better.
`Original` is fine too though on second thought because quality and format optimizations always happen by default anyway. Both would be okay to go with.
 
<img width="817" alt="image" src="https://github.com/imagekitio/docs-keep-a-changelog/assets/19747413/b10cfbc5-ee95-4cf1-8bdb-ec6c06768263">

Reply:
Cool, let's keep it as-is then.

Reply:
resolved